doctr/models/utils.py

```diff
@@ -112,10 +111,11 @@ class IntermediateLayerGetter(Model):
         model: the model to extract feature maps from
         layer_names: the list of layers to retrieve the feature map from
     """
+
```

Comment:
let's avoid modifying files that aren't related to this PR :)

Reply:
Sorry about that !